docs/diginstroom/sip/2.1/profiles/film.md

```diff
@@ -27,7 +28,7 @@ root_directory
 ├── METS.xml
 ├── metadata
 │   ├── descriptive
-│   │   └── dc+schema.xml
+│   │   └── dc_schema.xml
```

Comment:
Specific reason to change it (but not the `basic` and `material artwork`)?

Reply:
I changed it because using a plus sign resulted in errors when validating the sample with the Commons IP validator.

In any case, an underscore might be preferable regarding processing/parsing?

Reply:
I changed it back to a plus sign; it is indeed the validator that should be updated to handle plus signs correctly.